<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsBasicNumericFormatWidgetBase</class>
 <widget class="QgsPanelWidget" name="QgsBasicNumericFormatWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Round to</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QRadioButton" name="mRadDecimalPlaces">
     <property name="text">
      <string>Decimal places</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="mShowPlusCheckBox">
     <property name="text">
      <string>Show plus sign</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="mShowThousandsCheckBox">
     <property name="text">
      <string>Show thousands separator</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Decimal separator</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QgsFilterLineEdit" name="mDecimalLineEdit">
     <property name="maxLength">
      <number>1</number>
     </property>
     <property name="placeholderText">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="mDecimalsSpinBox">
     <property name="value">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QRadioButton" name="mRadSignificantFigures">
     <property name="text">
      <string>Significant figures</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="mShowTrailingZerosCheckBox">
     <property name="text">
      <string>Show trailing zeros</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QgsFilterLineEdit" name="mThousandsLineEdit">
     <property name="maxLength">
      <number>1</number>
     </property>
     <property name="placeholderText">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Thousands separator</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsPanelWidget</class>
   <extends>QWidget</extends>
   <header>qgspanelwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgsfilterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
